static/app/components/stream/groupChart.tsx

```diff
@@ -19,70 +19,49 @@ function asChartPoint(point: [number, number]): {name: number | string; value: n
 const EMPTY_STATS: ReadonlyArray<TimeseriesValue> = [];
 
 type Props = {
-  data: Group;
-  statsPeriod: string;
+  stats: ReadonlyArray<TimeseriesValue>;
   height?: number;
+  secondaryStats?: ReadonlyArray<TimeseriesValue>;
   showMarkLine?: boolean;
   showSecondaryPoints?: boolean;
 };
 
 function GroupChart({
-  data,
-  statsPeriod,
-  showSecondaryPoints = false,
+  stats,
   height = 24,
+  secondaryStats = EMPTY_STATS,
+  showSecondaryPoints = false,
   showMarkLine = false,
 }: Props) {
-  const stats: ReadonlyArray<TimeseriesValue> = statsPeriod
-    ? data.filtered
-      ? data.filtered.stats?.[statsPeriod]
-      : data.stats?.[statsPeriod]
-    : EMPTY_STATS;
-
-  const secondaryStats: TimeseriesValue[] | null =
-    statsPeriod && data.filtered ? data.stats[statsPeriod] : null;
```

Comment:
This was previously the logic to extract the stats from the group. As mentioned in the above comment, this is now delegated to the parent to limit the scope of this component.